index.html

```diff
@@ -4646,6 +4663,41 @@ <h2><span class="chunk">bKGD</span> Background color</h2>
           If the image bit depth is less than 16, the least significant bits are used.
           Encoders should set the other bits to 0, and decoders must mask the other bits to 0 before the value is used.</p>
         </section>
+
+        <section id="caBX">
+          <h2><span class="chunk">caBX</span> Content Credentials</h2>
+
+          <p>The four-byte chunk type field contains the hexadecimal values</p>
+
+          <pre>
+          63 61 42 58
+          </pre>
+
+          <p>The <span class="chunk">caBX</span> chunk contains content credentials
+            (provenance, and edit history) metadata 
+            in a secure, tamper-evident (cryptographically verifiable) 
+            and standardized way 
+            to enable publishers and consumers 
+            to determine the authenticity of media.
```

Comment:
```suggestion
            to determine the authenticity of media.
            It can also be used to declare whether a given image
            was created or edited by Generative AI or a human 
            or a combination of same.
```